data_adapter_oemof/adapter_classes.py

```diff
@@ -0,0 +1,37 @@
+from typing import Sequence, Union
+from dataclasses import dataclass
+
+
+@dataclass
+class VolatileAdapter:
```

Comment:
Once https://github.com/oemof/oemof-tabular/pull/44 is ready, the adapters could also be derived from the facades.